src/hotspot/share/utilities/vmError.cpp

```diff
@@ -1358,7 +1359,34 @@ void VMError::report_and_die(Thread* thread, const char* filename, int lineno, s
 
 void VMError::report_and_die(int id, const char* message, const char* detail_fmt, va_list detail_args,
                              Thread* thread, address pc, void* siginfo, void* context, const char* filename,
-                             int lineno, size_t size)
+                             int lineno, size_t size) {
+  report_and_maybe_die_impl(AfterReportAction::Die,
+                            id, message, detail_fmt, detail_args,
+                            thread, pc, siginfo, context,
+                            filename, lineno, size);
+  // Shouldn't be able to get here.  Die immediately if we do.
+  os::die();
```

Comment:
Should we try to report this somehow?

Reply:
I don't really see how, without being significantly at risk of recursive errors.

Reply:
tty->print_cr or even UL if possible?